### PR TITLE
Split the objective metric from the actual covered metric

### PR DESCRIPTION
**Pull request by dstallenberg**

### CODE REVIEW

File: libraries/search/lib/objective/managers/StructuralObjectiveManager.ts [hunk 1]
```diff
@@ -69,13 +78,10 @@ export class StructuralObjectiveManager<
       this._uncoveredObjectives.add(objective);
 
     // Set the current objectives
-    const rootObjectiveNodes = this._subject.cfg.functions.map(
-      (g) => g.graph.getChildren(g.graph.entry.id)[0] // should always be one child of the entry node
+    const rootObjectiveIds = this._subject.cfg.functions.map(
+      (g) => g.id // should always be one child of the entry node
```
Comment (mitchellolsthoorn) [inline libraries/search/lib/objective/managers/StructuralObjectiveManager.ts:82]: If I understand this correctly, this gives back the Id of the start node?

File: tools/base-language/lib/Metrics.ts [hunk 2]
```diff
@@ -18,32 +18,44 @@
 import { Metric, MetricType, SeriesType } from "@syntest/metric";
 
 export enum SeriesName {
-  PATHS_COVERED = "paths-covered",
   BRANCHES_COVERED = "branches-covered",
-  EXCEPTIONS_COVERED = "exceptions-covered",
+  STATEMENTS_COVERED = "statements-covered",
   FUNCTIONS_COVERED = "functions-covered",
-  LINES_COVERED = "lines-covered",
```
Comment (mitchellolsthoorn) [inline tools/base-language/lib/Metrics.ts:25]: Lines and paths can also be calculated for the actual coverage numbers right?

File: tools/base-language/lib/Metrics.ts [hunk 3]
```diff
@@ -18,32 +18,44 @@
...
-  IMPLICIT_BRANCHES_COVERED = "implicit-branches-covered",
-  OBJECTIVES_COVERED = "objectives-covered",
 
-  PATHS_TOTAL = "paths-total",
   BRANCHES_TOTAL = "branches-total",
-  EXCEPTIONS_TOTAL = "exceptions-total",
+  STATEMENTS_TOTAL = "statements-total",
   FUNCTIONS_TOTAL = "functions-total",
-  LINES_TOTAL = "lines-total",
```
Comment (mitchellolsthoorn) [inline tools/base-language/lib/Metrics.ts:45]: Same here

File: tools/base-language/lib/Metrics.ts (hunk 2 above)
Comment (dstallenberg) [reply tools/base-language/lib/Metrics.ts:25]: Yes, but the current instrumentation does not support this so I left them out.

File: libraries/search/lib/objective/managers/StructuralObjectiveManager.ts (hunk 1 above)
Comment (mitchellolsthoorn) [reply libraries/search/lib/objective/managers/StructuralObjectiveManager.ts:82]: Okay, sounds good

**Review by mitchellolsthoorn — APPROVED** [review]

**Pull request merged**